package.json

```diff
@@ -146,7 +146,7 @@
     "publish-preminor": "node scripts/clear-package-dir preminor --verbose && lerna publish preminor --pre-dist-tag=next --preid=next --force-publish --allow-branch=master --message=\"chore(release): Publish next pre-minor\"",
     "publish-next": "node scripts/clear-package-dir prerelease --verbose && lerna publish prerelease --pre-dist-tag=next --preid=next --allow-branch=master --message=\"chore(release): Publish next\"",
     "publish-rc": "node scripts/clear-package-dir prerelease --verbose && lerna publish prerelease --pre-dist-tag=rc --preid=rc --message=\"chore(release): Publish rc\"",
-    "publish-release": "node scripts/clear-package-dir patch --verbose && lerna publish patch",
+    "publish-release": "node scripts/clear-package-dir patch --verbose && lerna publish patch --dist-tag=latest-v2",
```

Comment:
Not sure about convention for that - but this is following one we had for v1 (`latest-vX`)